app/controllers/assignments_controller.rb

```diff
@@ -321,6 +321,8 @@ def is_int(val)
   end
 
   def set_questions_graders
+    # FIXME: This is way too complicated.
+
```

Comment:
Maybe.  I don't understand what happens to code in lib, but I do know that Rails doesn't automatically pick up changes to code there (I have to restart the server).  This was a pain while I was developing this code...